File: langs/C#/FizzBuzz.cs
using System;

class FizzBuzz
{
  static void Main()
  {
    var input = System.Console.ReadLine();
    int x = System.Int32.Parse(input);

    for (int i = 1; i <= x; i++)
    {
      if (i % 15 == 0)
        Console.Write("FizzBuzz");
      else if (i % 3 == 0)
        Console.Write("Fizz");
      else if (i % 5 == 0)
        Console.Write("Buzz");
      else
        Console.Write(i);

      Console.Write("\t");
    }
  }
}

File: langs/Rust/FizzBuzz.rs
use std::io;

fn main() {
  let mut input = String::new();
  io::stdin().read_line(&mut input).ok();
  let num: usize = input.trim().parse().ok().unwrap();

  for n in 1..(num + 1) {
    if n % 15 == 0 {
      print!("FizzBuzz");
    } else if n % 3 == 0 {
      print!("Fizz");
    } else if n % 5 == 0 {
      print!("Buzz");
    } else {
      print!("{}", n);
    }
    
    print!("\t");
  }
}
